Mlem/Views/Shared/Comments/Components/CommentBodyView.swift

```diff
@@ -95,14 +96,25 @@ struct CommentBodyView: View {
     func compactScoreDisplay() -> some View {
         let myVote = commentView.myVote ?? .resetVote
```

Comment:
any reason we couldn't pull this out into a computed property and ditch the weird return below?

Reply:
That would be much better, fixed in latest commit